tracing/trace_keyspace_helper.hh

```diff
@@ -142,6 +142,15 @@ private:
      */
     static cql3::query_options make_session_mutation_data(const one_session_records& all_records_handle);
 
+    /**
+     * Create a mutation data for a new session record, compatible with "sessions" table from previous version
+     *
+     * @param all_records_handle handle to access an object with all records of this session
+     *
+     * @return the relevant cql3::query_options object with the mutation data
+     */
+    static cql3::query_options make_session_mutation_data_fallback(const one_session_records& all_records_handle);
+
```

Comment:
I think you can use named bind parameters and have just a single query_options thing.

statement: INSERT INTO blah values (blah, blah, username) VALUES (:blah1, :blah2, :username)
fallback:  INSERT INTO blah values (blah, blah) VALUES (:blah1, :blah2)

The :username parameter in the query_options should just be ignored.